.cirrus.yml

```diff
@@ -0,0 +1,14 @@
+env:
+  CIRRUS_WORKING_DIR: "/var/tmp/go/src/github.com/containers/dnsname"
+  CIRRUS_SHELL: "/bin/bash"
+  GOPATH: "/var/tmp/go"
+  PATH: "$PATH:/var/tmp/go/bin"
```

Comment:
Not sure that will do what you want.  I think it will end up setting `PATH=:/var/tmp/go/bin` but I could be wrong.  If you want to set `$PATH` I would do it in the script directly.  So:

```YAML
    test_script:
        - export PATH=$PATH:$GOPATH/bin:$CIRRUS_WORKING_DIR/bin
        ...
```

Reply:
it seemed to work because make is being called and that doesnt exist in /var/tmp/go/bin

Reply:
ok.